modules/src/ics07_tendermint/header.rs

```diff
@@ -25,3 +25,46 @@ impl crate::ics02_client::header::Header for Header {
         self.signed_header.header.height()
     }
 }
+
+#[cfg(test)]
+pub mod test_util {
+    use crate::ics07_tendermint::header::Header;
+    use subtle_encoding::hex;
+    use tendermint::block::signed_header::SignedHeader;
+    use tendermint::validator::Info as ValidatorInfo;
+    use tendermint::validator::Set as ValidatorSet;
+    use tendermint::{vote, PublicKey};
+
+    // TODO: This should be replaced with a ::default() or ::produce().
+    // The implementation of this function comprises duplicate code (code borrowed from
+    // `tendermint-rs` for assembling a Header).
+    // See https://github.com/informalsystems/tendermint-rs/issues/381.
+    pub fn get_dummy_header() -> Header {
+        // Build a SignedHeader from a JSON file.
+        let shdr = serde_json::from_str::<SignedHeader>(include_str!(
+            "../../tests/support/signed_header.json"
```

Comment:
did you `git add` this file in ibc-rs? Can't see it here.

Reply:
Good point!